### DISCUSSION TITLE
Welcome to DeskThing-Pi Discussions!

### DISCUSSION THREAD

**Discussion by TIMBLOCKER** [TIMBLOCKER/DeskThing-Pi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VeryRandomness** [comment]
Howdy, I’m VeryRandomness. Currently a med student in Utah and I want to get started tinkering with tech. I wanted to make my own desk thing like in this video https://youtu.be/uQYIAYa27ds?si=GtOXFz6LoQE06LH2 but also want to have support for the four buttons on the screen. How could I add that support? I also want to add a rotary encoder in the future to control volume of playback, is that possible?

I don’t currently have the components yet, but will order them soon.

**Reply by TIMBLOCKER** [reply]
Hi @VeryRandomness,
no problem, you can always tinker with the project and give some updates on your progress. I'm currently writing my bachelor thesis so there is not a lot of energy working on the project. Just start with the repos and also look for the diferent branches. No need to feel stressed. 

Greetings from Germany,
TIMBLOCKER

**Reply by VeryRandomness** [reply]
Thanks! I will update you! Would I edit things in the defaultmapping.ts files to do what I want? Don't stress on responding fast! Good luck on your bachelor thesis!
